assets/src/edit-story/components/form/dateTime/datePicker.js

```diff
@@ -0,0 +1,109 @@
+// eslint-disable-next-line eslint-comments/disable-enable-pair
+/* eslint-disable header/header */
+/* Disable reason: This file is based on Gutenberg Datepicker */
```

Comment:
Note that the files in `dateTime` are currently mainly based on the Gutenberg dateTime picker. We might want to leave improvements/customizing this for a separate PR.